location/utils.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+    utils
+    ~~~~~
+
+    Utility functions for motif analysis.
+"""
+
+import pandas as pd
+import numpy as np
+import math
+from copy import deepcopy
+from collections import Counter
+from geopy.distance import vincenty
+import geohash
+
+from location import motif
+
+
+def approx_home_location(data, sr_col='stay_region'):
```

Comment:
You are right we could just use get_primary_location. I will change that in the codes.